components/script/dom/readablestream.rs

```diff
@@ -129,6 +129,7 @@ impl ReadableStream {
                 global,
                 UnderlyingSourceType::Js(underlying_source_dict),
                 extract_high_water_mark(strategy, 1.0)?,
+                extract_size_algorithm(strategy),
```

Comment:
Could you take this chance to expand the comments on this method?
I wanted to replace those step 1, step 2 to more useful spec comments for a while.